### ISSUE TITLE
Cannot read property 'id' of undefined when I click on submit button

### ISSUE THREAD

**Issue by Linda-Ikechukwu** [remark-ninja/remark-ninja-react#6]
I'm trying out the remark for a tutorial I'm writing on gatsby comments. 
After following the instructions on the docs to set it up, I can't submit comments.

I type in a comment, click on the submit button and then I get this error in the browser console: 

```
Uncaught TypeError: Cannot read property 'id' of undefined
    at onClick (index.js:183)
    at Object.o (react-dom.production.min.js:14)
    at p (react-dom.production.min.js:14)
    at react-dom.production.min.js:14
    at g (react-dom.production.min.js:15)
    at at (react-dom.production.min.js:52)
    at lt (react-dom.production.min.js:51)
    at ot (react-dom.production.min.js:52)
    at pt (react-dom.production.min.js:56)
    at L (react-dom.production.min.js:287)
```

I click on source and here's the line it points to: 

```
 lsSet(lsAuthorEmailKey, authorEmail);
    setError(undefined);
    setMessage(undefined);
    postComment({
      siteId: siteId,
      threadSlug: threadSlug,
      authorName: authorName,
      authorEmail: authorEmail,
      body: body,
      replyToComment: replyingTo.id
    }).then(function () {
      setMessage((0, _text["default"])('submitted'));
      setBody('');
```

**Comment by hjiang** [comment]
Are you using the latest version(2.4.1) from NPM? Is your site public? Is there a URL I can try?

**Comment by hjiang** [comment]
@Linda-Ikechukwu Apologies. I was able to reproduce it. It was indeed a bug. Could you please try the new version (2.4.2)? It should be fixed by 812db0a7594a4743975a06b9938455adc52ccf0d.

**Comment by Linda-Ikechukwu** [comment]
Yeah. It's fixed now
